pkgs/applications/kde/kpublictransport.nix

```diff
@@ -10,4 +13,10 @@ mkDerivation {
     maintainers = [ maintainers.samueldr ];
```

Comment:
same with this meta

Reply:
I reordered the new packages, I'd rather keep existing ones as they are for diffability and clean up in a separate change.